internal/fuse/snapshots_dir.go

```diff
@@ -66,6 +66,41 @@ func NewSnapshotsDir(root *Root, inode uint64, snapshots restic.Snapshots) *Snap
 	return d
 }
 
+// NewSnapshotsIDSDir returns a new directory containing snapshots named by ids.
+func NewSnapshotsIDSDir(root *Root, inode uint64, snapshots restic.Snapshots) *SnapshotsDir {
+	debug.Log("create snapshots ids dir with %d snapshots, inode %d", len(snapshots), inode)
+	d := &SnapshotsDir{
+		root:      root,
+		inode:     inode,
+		snapshots: snapshots,
+		names:     make(map[string]*restic.Snapshot, len(snapshots)),
+	}
+
+	// Track latest Snapshot
+	var latestTime time.Time
+	d.latest = ""
+
+	for _, sn := range snapshots {
+		name := fmt.Sprintf("%.8s", sn.ID().String())
```

Comment:
Did you discover `ID.Str()` yet? That's the method we're using everywhere to shorten an ID.

Reply:
Thank you